clang/test/Preprocessor/riscv-target-features.c

```diff
@@ -1120,6 +1121,14 @@
 // RUN:   -o - | FileCheck --check-prefix=CHECK-SMEPMP-EXT %s
 // CHECK-SMEPMP-EXT: __riscv_smepmp  1000000{{$}}
 
+// RUN: %clang --target=riscv32 -menable-experimental-extensions \
+// RUN: -march=rv32i_zalasr1p0 -x c -E -dM %s \
```

Comment:
Indent line continuations 2 spaces and drop `-x c`. This was recently updated in the reset of the file.